src/serial.c

```diff
@@ -858,6 +858,7 @@ static void evaluateCommand(void)
         mcfg.board_align_yaw = read16(); // board_align_yaw
         mcfg.currentscale = read16();
         mcfg.currentoffset = read16();
+        mcfg.motor_pwm_rate = read16();

```

Comment:
i added support for those 2 extra bytes in my dev branch for now, i have an idea how i could deal with such "non-compatible" changes in the future (where configurator wouldn't need a change of code where such thing gets added, and it would gracefully relay the data back afterwards even when there is no GUI support for the extra bytes), but it would take a good day or two of coding to rewrite the whole MSP to deal with data in that manner and it would also break the backup/restore implementation, so i am not so sure about committing to this just yet :/